### PR TITLE
Atari discovery draft.

### PR DESCRIPTION
**Pull request by fchirica**

### CODE REVIEW

File: chia/rpc/data_layer_rpc_api.py [hunk 1]
```diff
@@ -310,3 +314,25 @@ async def get_kv_diff(self, request: Dict[str, Any]) -> EndpointResult:
...
+        id_bytes = bytes32.from_hexstr(store_id)
+        urls = request["urls"]
+        amount = request["amount"]
+        fee = get_fee(self.service.config, request)
+        await self.service.add_mirror(id_bytes, urls, amount, fee)
+        return {}
+
+    async def delete_mirror(self, request: Dict[str, Any]) -> EndpointResult:
+        coin_id = request["id"]
+        id_bytes = bytes32.from_hexstr(coin_id)
```
Comment (altendky) [inline chia/rpc/data_layer_rpc_api.py:329]: Is this really the coin id?  Doesn't the coin id change with every update?

File: chia/rpc/data_layer_rpc_api.py [hunk 2]
```diff
@@ -310,3 +314,25 @@ async def get_kv_diff(self, request: Dict[str, Any]) -> EndpointResult:
...
+        fee = get_fee(self.service.config, request)
+        await self.service.delete_mirror(id_bytes, fee)
+        return {}
+
+    async def get_mirrors(self, request: Dict[str, Any]) -> EndpointResult:
+        store_id = request["id"]
+        id_bytes = bytes32.from_hexstr(store_id)
+        mirrors: List[Mirror] = await self.service.get_mirrors(id_bytes)
+        return {"res": [mirror.to_json_dict() for mirror in mirrors]}
```
Comment (altendky) [inline chia/rpc/data_layer_rpc_api.py:338]: Maybe?

```suggestion
        return {"mirrors": [mirror.to_json_dict() for mirror in mirrors]}
```

File: chia/rpc/data_layer_rpc_api.py (hunk 1 above)
Comment (Quexington) [reply chia/rpc/data_layer_rpc_api.py:329]: It's the coin ID of the mirror.  Each new mirror creation creates a coin ID to use to delete it.  Does matter what the singleton does in the meantime.

**Review by Quexington — APPROVED** [review]
Wallet related stuff looks good

**Pull request merged**